tests/Findologic/Response/Xml21ResponseParserTest.php

```diff
@@ -169,52 +169,52 @@ public function filterResponseProvider(): array
 
         $expectedVendorFilter = new VendorImageFilter('vendor', 'Hersteller');
         $expectedVendorFilter->addValue(
-            (new ImageFilterValue('Anderson, Gusikowski and Barton', 'Anderson, Gusikowski and Barton'))
+            (new ImageFilterValue('Anderson, Gusikowski and Barton', 'Anderson, Gusikowski and Barton', 'vendor'))
```

Comment:
Set the filter names to a variable. This way if we change it in the future you do not have to change it manually multiple times in the same methods.